test/Sema/bitwse_copyable_import.swift

```diff
@@ -85,3 +90,7 @@ func passIntsTrailing2(_ t: IntsTrailing2) {
 func passIntsTrailing3(_ t: IntsTrailing3) {
   take(t)
 }
+
+func passE(_ e: E) {
+  take(e)
```

Comment:
What is this testing?

Reply:
That an imported enum E conforms to BitwiseCopyable.